GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -208,6 +209,7 @@ private bool IsGVFSUpgradeAllowed(out string consoleError)
                     Environment.NewLine,
                     "GVFS Service is not running.",
                     adviceText);
+                this.tracer.RelatedError($"{nameof(this.IsGVFSUpgradeAllowed)}: Upgrade is not installable. {consoleError}");

```

Comment:
Same feedback here, should this be an error logged to telemetry?

Reply:
As discussed, changing this to warning.